src/main/java/Duke.java

```diff
@@ -1,10 +1,29 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        // greeting message
+        String greetings = "Hi there! 👋 I'm Duke\n"
+                            + "What can I do for you?";
+
+        System.out.println(greetings);
+
+        // reading user input
+        Scanner sc = new Scanner(System.in);
+
+        // boolean to indicate whether userInput is bye
+        boolean isBye = false;
+
+        String userInput = sc.nextLine();
+
+        while (!isBye) {
+            if (userInput.equals("bye")) {
+                isBye = true;
+                System.out.println("Bye. See you again next time! Have a nice day 😊!");
```

Comment:
Would be great if you could do what you did earlier with the greeting String and line-break and + here!